src/lib/components/DisclaimerModal.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+	import { base } from "$app/paths";
+	import { page } from "$app/stores";
+	import {
+		PUBLIC_APP_DATA_SHARING,
+		PUBLIC_APP_DESCRIPTION,
+		PUBLIC_APP_NAME,
+	} from "$env/static/public";
+	import LogoHuggingFaceBorderless from "$lib/components/icons/LogoHuggingFaceBorderless.svelte";
+	import Modal from "$lib/components/Modal.svelte";
+	import type { LayoutData } from "../../routes/$types";
+	import Logo from "./icons/Logo.svelte";
+
+	export let settings: LayoutData["settings"];
+</script>
+
+<Modal>
+	<div
+		class="from-primary-500/40 via-primary-500/10 to-primary-500/0 flex w-full flex-col items-center gap-6 bg-gradient-to-b px-5 pb-8 pt-9 text-center sm:px-6"
+	>
+		<h2 class="flex items-center text-2xl font-semibold text-gray-800">
+			<Logo classNames="mr-1" />
+			{PUBLIC_APP_NAME}
+		</h2>
+
+		<p class="text-lg font-semibold leading-snug text-gray-800" style="text-wrap: balance;">
+			{PUBLIC_APP_DESCRIPTION}
+		</p>
+
+		<p class="text-sm text-gray-500">
+			Disclaimer: AI is an area of active research with known problems such as biased generation and
+			misinformation. Do not use this application for high-stakes decisions or advice.
```

Comment:
```suggestion
			Disclaimer: AI can produce biased generation and
			misinformation. Do not use this application for high-stakes decisions or advice.
```
nit: I think we can make this description shorter

Reply:
I think the exact text of the modal was decided internally with @meg-huggingface, I wouldn't change it for now but we can always update it later if we agree on something

Reply:
Oh I see 👍